src/content/app/tools/blast/components/blast-hits-diagram/BlastHitsDiagram.scss

```diff
@@ -0,0 +1,9 @@
+@import 'src/styles/common';
+
+.backbone {
+  fill: $medium-dark-grey;
```

Comment:
It might very well be; but Andrea said medium-dark-grey in the ticket. It looks lighter in the XDs, for sure. I don't have a strong opinion whether to follow her instruction and wait for her to make adjustments if necessary, or to adjust ourselves to our liking.